web/src/server_events_dispatch.js

```diff
@@ -680,7 +674,6 @@ export function dispatch_normal_event(event) {
                             message_lists.current.update_trailing_bookend(true);
```

Comment:
Also, currently the bookend is updated to "This channel has been archived" when user loses access to a stream and this has been the behavior originally. Do we want to change it?

Reply:
The archived channel bookend seems fine to me? It doesn't seem important whether your'e subscribed if no new messages can be added.

Reply:
But the stream is not archived, it is just that user cannot access the stream.

Reply:
Ahh, OK. Let's start a #design thread; probably we want to have you sent to your home view when you lose access to a channel you're currently viewing.